src/applications/Task.java

```diff
@@ -19,4 +19,7 @@ public int getMachine() {
     public int getTime() {
         return time;
     }
+    public static int getNumTasks(SimulationSpecification specification, int i) {
```

Comment:
This makes me wonder if what you want is a `JobSpecification` class that allows you to ask a instance of that class how many tasks it has? 🤔

Reply:
This will likely be something I will be addressing in the future. I agree that this doesn't seem to be in the right place.